docs/dex-specification.md

```diff
@@ -656,6 +656,8 @@ Long-term staking allows users to stake the Dex token and earn other tokens as r
 
 The contract deploys a new staking pool contract using [`deployPool`](#deploypool). The created contract is owned by the [`multisig` contract](#multisignature-wallet).
 
+The `StakingFactory` contract is responsible for deploying new [`StakingInitializable`](#stakinginitializable) contracts. While the factory fully initializes the new contract, the reward tokens have to be sent manually after the deployment.
```

Comment:
Pool initialization happens after the actual deployment in the deployPool function.